Software/Python/ir_remote_control/server/install.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
```

Comment:
How does this un-install monit?  If systemd and the monit are both running, will there be collisions?